Sources/PrinceOfVersions/ResponseModels/UpdateData/UpdateInfo.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  UpdateInfo.swift
+//  PrinceOfVersions
+//
+//  Created by Jasmin Abou Aldan on 06/07/16.
+//
+//
+
+import Foundation
+
+#if canImport(UIKit)
+import UIKit
+#elseif canImport(AppKit)
+import AppKit
+#endif
+
+// MARK: - Internal configuration data -
+
+public struct UpdateInfo: Decodable {
+
+    // MARK: - Private properties -
+
+    private var ios: [ConfigurationData]?
+    private var ios2: [ConfigurationData]?
```

Comment:
what does the 2 stand for?

Reply:
@Truba 
Since older versions of PoV (<4.0) are not compatible with new versions (>=4.0), this is a solution for letting developers update PoV but still keeping JSON compatible with older PoV version. 

To support older versions of PrinceOfVersions (<4.0) and new versions (>=4.0) it is important to specify appropriate configurations under the correct key. The older version will be stored in `ios` property and the new one in `ios2` property. The reason behind naming is to match JSON file. ( See example at https://pastebin.com/raw/0MfYmWGu ). 

If JSON only has support for new PoV versions (>=4.0), configuration data will be stored in `ios` property and `ios2` will be ignored.

Reply:
Ty, can you please add some of that text you just wrote into the comment in this file? It could be very useful for someone like me who doesn't know what is going on?